bin/pubber

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-ROOT_DIR=$(realpath $(dirname $0)/..)
+ROOT_DIR="$(pwd)"
```

Comment:
this won't work since there shouldn't be a requirement to run it from a certain directory. It needs to be in reference to $0, since that specifies the target (udmi install directory), not the directory it's run from.